src/kernels/helpers.ts

```diff
@@ -1975,10 +1977,10 @@ export async function handleKernelError(
     return resultController;
 }
 
-function convertContextToFunction(context: 'start' | 'interrupt' | 'restart') {
+function convertContextToFunction(context: 'start' | 'interrupt' | 'restart', options?: IDisplayOptions) {
     switch (context) {
         case 'start':
-            return (k: IKernel) => k.start();
+            return (k: IKernel) => k.start(options);
```

Comment:
Keep track of whether UI is to be displayed or not.
When we use auto start we need to go through his same code.